roles/foreman_installer/tasks/module_pr.yml

```diff
@@ -32,3 +32,13 @@
   with_items: "{{ foreman_installer_module_prs.split(',') }}"
   tags:
     - packages
+
+- name: 'Trust github'
+  blockinfile:
```

Comment:
Would https://docs.ansible.com/ansible/latest/known_hosts_module.html make sense to use?